hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/DroolsRulesEngineImpl.java

```diff
@@ -182,6 +182,12 @@ public void fire() {
                     }
                 }
             }
+            long filteringTime = System.currentTimeMillis() - startFiltering;
+            log.debugf("Filtering BatchData [%s] took [%s]", batchData.size(), filteringTime);
+            if (filteringTime > PERF_FILTERING_THRESHOLD) {
+                log.warnf("Filtering Filtering BatchData [%s] took [%s] ms exceeding [%s] ms",
```

Comment:
> Duplicate 'Filtering' in message (also see comment about changing 'Filtering' terminology).

Well, one is on debug and other in warning, content message is similar but not equals.